src/platform/apollolake/include/platform/memory.h

```diff
@@ -250,7 +250,7 @@
 
 #define SOF_TEXT_START		(HP_SRAM_VECBASE_RESET + 0x400)
 #define SOF_TEXT_BASE		(SOF_TEXT_START)
-#define SOF_TEXT_SIZE		(0x1a000 - 0x400 + 0x1000)
+#define SOF_TEXT_SIZE		(0x1a000 - 0x400 + 0x2000)
```

Comment:
Probably better to say 0x1c000 - 0x400.  And it would also be good if we had a macro from the 0x400.